RocketChat/Rocket.Chat
# chore: fetch all slashcommands on startup

## Proposed changes (including videos or screenshots)


Not all apps were being fetched since we would just get the default pagination size, so we would fetch only 50 apps, now we will search all the apps in order to have everything once the user is typing

## Issue(s)


## Steps to test or reproduce

Use a workspace with over 50 slashcommands, just check if all of them are available

## Further comments

[CONN-111](https://rocketchat.atlassian.net/browse/CONN-111)

[CONN-111]: https://rocketchat.atlassian.net/browse/CONN-111?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
